### PR TITLE
[docs] fix kubernetes operations anchor link

### PR DESCRIPTION
**Pull request by judithpatudith**
Fix Kubernetes operations anchor link.

### CODE REVIEW

File: website/source/docs/guides/index.html.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Tracks include:
 - [Getting Started with Kubernetes](https://learn.hashicorp.com/consul/?track=getting-started-k8s#getting-started-k8s)
 - [Day 1: Deploying Your First Datacenter](https://learn.hashicorp.com/consul/?track=datacenter-deploy#datacenter-deploy)
 - [Day 1: Security and Network Operations](https://learn.hashicorp.com/consul/?track=security-networking#security-networking)
-- [Day 1: Kubernetes Operations](https://learn.hashicorp.com/consul/?track=kubernetes#kubernetes)
+- [Day 1: Kubernetes Operations](https://learn.hashicorp.com/consul/?track=day-1-operations#day-1-operations)
```
Comment (pearkes) [inline website/source/docs/guides/index.html.md]: When I go to this I can't actually see the intended content as it is above the anchor. Is there a way to get around that? Could we link to a specific guide?

Comment (judithpatudith) [reply website/source/docs/guides/index.html.md]: We could link to the first guide in each track. @kaitlincarter-hc do you have thoughts?

Comment (judithpatudith) [reply website/source/docs/guides/index.html.md]: Talked this over with Kait: Not all of the tracks have index pages and the top level guides page is easy to navigate, so we decided to replace the links to specific tracks with a CTA pointing to the learn platform instead. We can add track links back once the anchor links get fixed

Comment (pearkes) [reply website/source/docs/guides/index.html.md]: 👍

Comment (pearkes) [reply website/source/docs/guides/index.html.md]: 👍

**Pull request merged**